src/libs/actions/Session.js

```diff
@@ -23,11 +23,9 @@ Onyx.connect({
  */
 function setSuccessfulSignInData(data) {
     PushNotification.register(data.accountID);
-    Onyx.multiSet({
-        [ONYXKEYS.SESSION]: {
-            shouldShowComposeInput: true,
-            ..._.pick(data, 'authToken', 'accountID', 'email'),
-        },
+    Onyx.merge(ONYXKEYS.SESSION, {
+        shouldShowComposeInput: true,
+        ..._.pick(data, 'authToken', 'accountID', 'email'),
```

Comment:
Yeah I was thinking about that, my assumption is that since it was done 7 months ago, we just didn't "need to" do `.merge` yet. I can keep an eye on issues related to Session storage in the next few weeks just in case